internal/controller/http/v1/profiles.go

```diff
@@ -126,8 +126,9 @@ func (r *profileRoutes) getByName(c *gin.Context) {
 
 func (r *profileRoutes) export(c *gin.Context) {
 	name := c.Param("name")
+	domainName := c.Query("domainName")
```

Comment:
looks like we're missing some API validation -- we should add that

Reply:
done